spring-boot-admin-server-ui/src/main/frontend/views/instances/details/details-memory.vue

```diff
@@ -90,55 +89,67 @@ export default {
         default:
           return this.type;
       }
-    }
+    },
   },
   methods: {
     prettyBytes,
     async fetchMetrics() {
-      const responseMax = this.instance.fetchMetric('jvm.memory.max', {area: this.type});
-      const responseUsed = this.instance.fetchMetric('jvm.memory.used', {area: this.type});
-      const hasMetaspace = (await responseUsed).data.availableTags.some(tag => tag.tag === 'id' && tag.values.includes('Metaspace'));
-      const responeMetaspace = this.type === 'nonheap' && hasMetaspace
-        ? this.instance.fetchMetric('jvm.memory.used', {area: this.type, id: 'Metaspace'})
-        : null;
-      const responseCommitted = this.instance.fetchMetric('jvm.memory.committed', {area: this.type});
+      const responseMax = this.instance.fetchMetric('jvm.memory.max', {
+        area: this.type,
+      });
+      const responseUsed = this.instance.fetchMetric('jvm.memory.used', {
+        area: this.type,
+      });
+      const hasMetaspace = (await responseUsed).data.availableTags.some(
+        (tag) => tag.tag === 'id' && tag.values.includes('Metaspace')
+      );
+      const responeMetaspace =
+        this.type === 'nonheap' && hasMetaspace
+          ? this.instance.fetchMetric('jvm.memory.used', {
+              area: this.type,
+              id: 'Metaspace',
+            })
+          : null;
+      const responseCommitted = this.instance.fetchMetric(
+        'jvm.memory.committed',
+        { area: this.type }
+      );
       return {
         max: (await responseMax).data.measurements[0].value,
         used: (await responseUsed).data.measurements[0].value,
-        metaspace: responeMetaspace ? (await responeMetaspace).data.measurements[0].value : null,
-        committed: (await responseCommitted).data.measurements[0].value
+        metaspace: responeMetaspace
+          ? (await responeMetaspace).data.measurements[0].value
+          : null,
+        committed: (await responseCommitted).data.measurements[0].value,
       };
     },
     createSubscription() {
       const vm = this;
-      return timer(0, sbaConfig.uiSettings.pollTimer.memory)
-        .pipe(concatMap(this.fetchMetrics), retryWhen(
-          err => {
-            return err.pipe(
-              delay(1000),
-              take(5)
-            )
-          }))
+      return timer(0, 1000)
+        .pipe(
+          concatMap(this.fetchMetrics),
+          retryWhen((err) => {
+            return err.pipe(delay(1000), take(5));
```

Comment:
I created this [issue](https://github.com/codecentric/spring-boot-admin/issues/3243) regarding this change.